projects/core/components/svg/svg.component.ts

```diff
@@ -48,7 +48,7 @@ const FAILED_EXTERNAL_ICON = 'Failed to load external SVG';
 export class TuiSvgComponent {
     private readonly src$ = new ReplaySubject<void>(1);
     private readonly isIE = isIE(this.userAgent);
-    private icon = '';
+    private icon: keyof typeof import('@taiga-ui/icons') | string = '';
```

Comment:
`keyof typeof import('@taiga-ui/icons')` seems so long. How about type alias?

Reply:
Where do you think it's best to place an alias? `kit` or `cdk` package or export from `icons` package?

Reply:
Hmm.. I think cdk is the best place

Reply:
Definitely not `cdk` and also this makes packages dependent. It would cause issues with proprietary icons because there are more names and because in proprietary environment we do not install public icons package. I think this whole PR is doubtful.